src/stache/includes/deprecation-warning.html

```diff
@@ -0,0 +1,3 @@
+<sky-alert alertType="danger">
+  This CLI and template have been deprecated.  Please visit <a href="https://developer.blackbaud.com/skyux-migration-guide">the migration guide</a> to learn how to upgrade.
```

Comment:
Do you want to link to here, or to the docs @Blackbaud-LindseyRix is working on?

Reply:
I would prefer the Stache update I would think.